server/sheet.go

```diff
@@ -0,0 +1,199 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/bytebase/bytebase/api"
+	"github.com/bytebase/bytebase/common"
+	"github.com/google/jsonapi"
+	"github.com/labstack/echo/v4"
+)
+
+func (s *Server) registerSheetRoutes(g *echo.Group) {
+	g.POST("/sheet", func(c echo.Context) error {
+		ctx := context.Background()
+		sheetCreate := &api.SheetCreate{}
+		if err := jsonapi.UnmarshalPayload(c.Request().Body, sheetCreate); err != nil {
+			return echo.NewHTTPError(http.StatusBadRequest, "Malformatted create sheet request").SetInternal(err)
+		}
+
+		if sheetCreate.Name == "" {
+			return echo.NewHTTPError(http.StatusBadRequest, "Malformatted sheet request, missing name")
+		}
+		if sheetCreate.Statement == "" {
```

Comment:
I guess we need to support empty statement since it's normally OK to have an empty doc?

Reply:
agree, started with an empty sheet.